WordPress/Classes/Models/Post.m

```diff
@@ -196,18 +196,19 @@ - (NSString *)contentPreviewForDisplay
 
 - (NSString *)statusForDisplay
 {
-    NSString *statusString = [self statusTitle];
-    if ([self.status isEqualToString:PostStatusPublish] || [self.status isEqualToString:PostStatusDraft]) {
-        statusString = nil;
-    }
-    if (![self hasRemote]) {
+    NSString *statusString = nil;
+    
+    if (self.isRevision) {
         NSString *localOnly = NSLocalizedString(@"Local", @"A status label for a post that only exists on the user's iOS device, and has not yet been published to their blog.");
         if (statusString) {
             statusString = [NSString stringWithFormat:@"%@, %@", statusString, localOnly];
         } else {
             statusString = localOnly;
```

Comment:
I think my expectation was  that rather than replacing the existing status completely, the "Local" would be an additional status.  For example, this is what currently is displayed when creating a scheduled post while offline.  Notice that both Scheduled, and Local are displayed.

<img width="379" alt="screen shot 2016-03-14 at 4 16 14 pm" src="https://cloud.githubusercontent.com/assets/1435271/13760262/3df40fd8-ea00-11e5-8304-f3d604762efa.png">

The changes in this branch replace the status completely with "local" which seems counter to our existing design. This is what I was referring to when I said it seemed problematic for certain post statuses. 

Maybe in the case of local edits to existing posts, it could continue to display whatever the status was (if its a status we display) and have "Local Changes" appended?